.yarn/patches/@metamask-previews-keyring-controller-npm-7.4.0-preview.f0f357a-50cc3f549f.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/dist/KeyringController.d.ts b/dist/KeyringController.d.ts
```

Comment:
To be removed

Reply:
Removed on commit 1819c35bef4b8d3628429686ddf3bacdcd3d4262